src/components/result/CategoryHeader/types.ts

```diff
@@ -1,6 +1,4 @@
-import type { GetPostsRes } from '@apis/post'
-
 export type ResultHeaderProps = {
   searchResult: string
-  postData?: GetPostsRes[]
+  postSummaries: number
```

Comment:
```suggestion
postSummariesLength: number
```
`postSummariesLength`가 더 명확할 것 같습니다!

Reply:
[완료](https://github.com/price-offer/offer-fe/pull/174/commits/07442cc6093ada90b9e21e15938cc48e34573066#diff-6056e8ce5d05fbdd1442f9bcc64669875f709d5fa2a625b4c5b9cae9acb24cfa)